temporalcli/commands.taskqueue.go

```diff
@@ -29,9 +29,20 @@ type pollerRowType struct {
 	RatePerSecond  float64   `json:"ratePerSecond"`
 }
 
+type statsRowType struct {
+	BuildID                 string        `json:"buildId"`
+	TaskQueueType           string        `json:"taskQueueType"`
+	ApproximateBacklogCount int64         `json:"approximateBacklogCount"`
+	ApproximateBacklogAge   time.Duration `json:"approximateBacklogAge"`
+	BacklogIncreaseRate     float32       `json:"backlogIncreaseRate"`
+	TasksAddRate            float32       `json:"tasksAddRate"`
+	TasksDispatchRate       float32       `json:"tasksDispatchRate"`
+}
+
 type taskQueueDescriptionType struct {
 	Reachability []taskQueueReachabilityRowType `json:"reachability"`
 	Pollers      []pollerRowType                `json:"pollers"`
+	Stats        []statsRowType                 `json:"stats"`
```

Comment:
What's the benefit of separate maps for build ID -> reachability, build ID -> poller, and build ID -> stats? isn't better to have a single map of build ID -> {reachability, pollers, stats}? cc @antlai-temporal @cretz

Reply:
I too, am not totally opposed to this idea - included the new field cause I thought a potential norm could have been to have separate maps for each but would love to hear your thoughts @cretz

Reply:
I have no strong opinions on the structure of the output (nor did I when this was first developed). Anything sane works for me. But just remember, it must be stable.

Reply:
It is just for modularity, these are very different top-level concepts let's keep them separate. The overhead of three maps is just minimal...